pkg/crc/preflight/preflight_darwin.go

```diff
@@ -13,52 +13,32 @@ import (
 )
 
 // SetupHost performs the prerequisite checks and setups the host to run the cluster
-func hyperkitPreflightChecks(networkMode network.Mode) []Check {
-	return []Check{
-		{
-			configKeySuffix:  "check-m1-cpu",
-			checkDescription: "Checking if running emulated on a M1 CPU",
-			check:            checkM1CPU,
-			fixDescription:   "CodeReady Containers is unsupported on Apple M1 hardware",
-			flags:            NoFix,
-
-			labels: labels{Os: Darwin},
-		},
-		{
-			configKeySuffix:  "check-hyperkit-installed",
-			checkDescription: "Checking if HyperKit is installed",
-			check:            checkHyperKitInstalled(networkMode),
-			fixDescription:   "Setting up virtualization with HyperKit",
-			fix:              fixHyperKitInstallation(networkMode),
-
-			labels: labels{Os: Darwin},
-		},
-		{
-			configKeySuffix:  "check-qcow-tool-installed",
-			checkDescription: "Checking if qcow-tool is installed",
-			check:            checkQcowToolInstalled,
-			fixDescription:   "Installing qcow-tool",
-			fix:              fixQcowToolInstalled,
+var vfkitPreflightChecks = []Check{
+	{
+		configKeySuffix:  "check-m1-cpu",
+		checkDescription: "Checking if running emulated on a M1 CPU",
+		check:            checkM1CPU,
+		fixDescription:   "CodeReady Containers is unsupported on Apple M1 hardware",
```

Comment:
`is currently unsupported`

Reply:
It would be better to reword this, this check only triggers when trying to run a x86_64 executable on a m1 cpu, which we'll never support.

Reply:
`This version of CodeReady Containers for x86_64 is unsupported ...`